packages/foundations-vars/src/_colours.css

```diff
@@ -0,0 +1,176 @@
+:root {
+  /* Colours */
+  --color-white: #ffffff;
+  --color-black: #000000;
+
+  /* Text greys */
+  --color-text: #282828;
+  --color-text-alt: #595959;
+
+  /* Grey backgrounds */
+  --color-grey-mid-light: #d8d8d8;
+  --color-grey-mid: #c4c4c4;
+  --color-grey-dark: #6e6e6e;
+
+  /* Neutral backgrounds */
+  --color-grey-neutral-light: #f3f3f3;
+  --color-grey-light: var(--color-grey-neutral-light);
+  --color-grey-neutral-warm: #f5f5f0;
+  --color-grey-neutral-cool: #dde9ed;
+  --color-grey-neutral-cool-light: #f2f4f7;
+
+  /* Links */
+  --color-link: #00729a;
+  --color-link-hover: #00394e;
+  --color-link-focus: #8d44d8;
+  --color-link-active: #002c3d;
+  --color-link-visited: var(--color-link-active);
+  /* Phasing out the '--' for modifiers as it causes issues with design token processing (EG lodash) */
+  --color-link--hover: var(--color-link-hover);
+  --color-link--focus: var(--color-link-focus);
+  --color-link--active: var(--color-link-active);
+  --color-link--visited: var(--color-link-active);
+
+  /* Buttons */
```

Comment:
is there a focus state for buttons?

Reply:
@gbbns Yeah but it just uses `--color-button-blue-hover`

These lines aren't new ones, just moved into new separate files.

The only new bits in this PR are found in **_colours-brands.css** if that helps?

Reply:
Ace, ta.